sections/07_conclusions.md

```diff
@@ -52,6 +52,23 @@ addition, the stochasticity and complexity of biological systems implies that
 for some problems, for instance, predicting gene regulation in disease, perfect
 accuracy will be unattainable.
 
+We are witnessing deep learning models achieving human-level performance across
+a number of biomedical domains, and yet we do not believe that biologists and
+clinicians will be out of a job anytime soon. While deep learning and other
+advanced algorithms will soon be (or already are) better than scientists at
+specific tasks, in many cases they would still be unable to fully grasp the
+bigger picture. Deep neural networks are also prone to mistakes that humans are
+much more unlikely to make, such as incorrectly classifying negative images
+[@arxiv:1703.06857], an important reminder that these algorithms do not really
+understand or "recognise the semantic of the object". Until true and reliable
+artificial intelligence becomes standard in the laboratory and in the clinic,
+the human factor still has a critical role to play. In some cases, cooperation
```

Comment:
I like raising this human factor in conclusion.  I tried to present this idea in the revised intro as well.